src/entities/auth-entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity()
+export class RefreshToken {
+  // eslint-disable-next-line no-useless-constructor,no-empty-function
+  public constructor(token: string) {
+    this.token = token;
+  }
+
+  @PrimaryGeneratedColumn('uuid')
+  public id: string;
+
+  @Column({ type: 'varchar' })
```

Comment:
Could I propose other schem?

At first, current  schem doesnt store user id, now id is just random uuid. As second we can use tokens as primary keys.

Reply:
User id is encrypted in token.

Reply:
I reply here too. 

How to find token by userId? to do this with current schem you need to:
1) fetch all tokens
2) parse all payloads
3) find needed tokens

Reply:
> User id is encrypted in token.

Let me correct, it's encoded) Encrypted is about [JWE](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwjWgvapmsjhAhWAw8QBHW4IAvsQFjAAegQIABAB&url=https%3A%2F%2Ftools.ietf.org%2Fhtml%2Frfc7516&usg=AOvVaw19Q8tBrm7mw3J3Lar3kpUP)